examples/failing/3132.purs

```diff
@@ -0,0 +1,18 @@
+-- @shouldFailWith TransitiveExportError
```

Comment:
Updated the reason this should fail.

Reply:
Can you paste the full error you get when you attempt to compile this?

Reply:
```
Failing examples
  '3132.purs' should fail with 'UnknownName' FAILED [1]

Failures:

  tests/TestCompiler.hs:218:
  1) Failing examples '3132.purs' should fail with 'UnknownName'
       Expected these errors: ["UnknownName"], but got these: ["TransitiveExportError"], full error messages:
       in module Main
       at /home/matt/Projects/purescript/examples/failing/3132.purs line 1, column 1 - line 18, column 7

         An export for class C3 requires the following to also be exported:

           class C1
           class C2



       See https://github.com/purescript/documentation/blob/master/errors/TransitiveExportError.md for more information,
       or to contribute content related to this error.
```

Reply:
Great, thank you. I was just wondering. :)

Reply:
"Does this fail with the right type of error, but also the right *reason* that I am expecting?" is definitely a good thing to check :smile: